e2e-pw/src/oss/poms/modal/leva.ts

```diff
@@ -0,0 +1,36 @@
+import { Locator, Page, expect } from "src/oss/fixtures";
+import { ModalPom } from ".";
+
+export class ModalLevaPom {
+  readonly page: Page;
+  readonly locator: Locator;
+  readonly assert: LevaAsserter;
+
+  constructor(page: Page, private readonly modal: ModalPom) {
+    this.page = page;
+    this.locator = page.locator("#fo-leva-container");
+    this.assert = new LevaAsserter(this);
+  }
+}
+
+class LevaAsserter {
+  constructor(private readonly modalLevaPom: ModalLevaPom) {}
+
+  async verifyDefaultFolders() {
+    const defaultFolderNames = ["Visibility", "Labels", "Lights"];
```

Comment:
Minor suggestion, but I find it useful to constants as define as a module global  after import, e.g. 
```typescript
const DEFAULT_FOLDER_NAMES = ["Visibility", "Labels", "Lights"]
```